engine/admin/multi-process_container.md

```diff
@@ -0,0 +1,89 @@
+---
+description: How to run more than one process in a container
+keywords: docker, supervisor, process management
+redirect_from:
+- /engine/articles/using_supervisord/
+- /engine/admin/using_supervisord/
+title: Run multiple processes in a container
+---
+
+A container's running process is usually the `ENTRYPOINT` and/or `CMD` at the
+end of the `Dockerfile`. It is generally recommended that you separate areas of
+concern by using one container per process, and connect those containers using
```

Comment:
We may want to tweak a bit w.r.t. "one process per container"; it may be a bit more nuanced; it's a single "service" per container. That service may fork into multiple _processes_ (for example, Apache starts multiple worker processes). It's ok to have multiple processes, but the concept of Docker is to not have a container be responsible for multiple services / tasks.

Slightly orthogonal: 

The container's main process is responsible for managing all processes that it started. In _some_ cases, the main process isn't well-designed, and doesn't handle "reaping" of child processes well when the container exists. If you're dealing with such a process, you can use the `--init` option, which inserts a tiny init-process in the container as the main process, and handles "reaping" of all processes when the container exits. There's no reason to use a full-fledged init process (Sysvinit, upstart, systemd) for that.